### PR TITLE
Fixes #2180 - Port capnp serialization to CLAClassifierRegion class

### LINKED ISSUE #2180: Port capnp serialization to CLAClassifieryRegion class
See parent issue: https://github.com/numenta/nupic/issues/2158

### PR DESCRIPTION
**Pull request by david-ragazzi**
Fixes https://github.com/numenta/nupic/issues/2180

blocked by #2242

### CODE REVIEW

**Comment by david-ragazzi** [comment]
@scottpurdy I'm using the default assertEqual function for this PR. So you can review independent of the discussion in https://github.com/numenta/nupic/pull/2197

File: nupic/regions/CLAClassifierRegion.capnp [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@0xf091b1bb874049df;
+
+using import "/nupic/bindings/proto/ClaClassifier.capnp".ClaClassifierProto;
```
Comment (scottpurdy) [inline nupic/regions/CLAClassifierRegion.capnp]: Please use relative paths for now (there is a bug in pycapnp that makes mixing relative and absolute paths cause problems)

Comment (david-ragazzi) [inline nupic/regions/CLAClassifierRegion.capnp]: > Please use relative paths for now (there is a bug in pycapnp that makes mixing relative and absolute paths cause problems)

@scottpurdy I saw a discussion between you and @oxtopus about this. Is this a final solution?

Comment (scottpurdy) [inline nupic/regions/CLAClassifierRegion.capnp]: Yes that is the final temporary solution :smile: 

#2214

**Comment by david-ragazzi** [comment]
> Please use relative paths for now (there is a bug in pycapnp that makes mixing relative and absolute paths cause problems)

Done. Can I merge?

**Comment by scottpurdy** [comment]
Please hold off on this PR until PyRegion serialization works. I will review it then.

**Comment by david-ragazzi** [comment]
> Please hold off on this PR until PyRegion serialization works. I will review it then.

Ok, no problems..

**Comment by david-ragazzi** [comment]
> Please hold off on this PR until PyRegion serialization works. I will review it then.

Is there some work related to this? Some newer PRs related to KNN and SP that are beeing merged are causing a lot of re-work to my PRs.

**Comment by scottpurdy** [comment]
@david-ragazzi - Any Python regions won't be useable until we tackle the PyRegion serialization. There may be other things that need to be done for the network API in Python but that is the only thing I am aware of so far. See #2241.

We could go ahead with code reviews and get the serialization for the regions merged before it can be used but will likely need to make some changes after the network level stuff is implemented.

I still have some work left on the C++ side for serialization. If you have the time, #2242 is the main blocking task.

**Pull request closed without merging**